.github/MAINTAINING.md

```diff
@@ -0,0 +1,168 @@
+  * [Test Coverage](#test-coverage)
+* [Bugs](#bugs)
+  * [Bug Management](#bug-management)
+* [Jira Tickets](#jira-tickets)
+* [Service](#service)
+* [Images](#images)
+
+
+# Overview
+
+This document is for documenting only the aspects of the Addon Operator that need maintenance. This is a living document and is meant to be iterated and enhanced as necessary. Furthermore, this document is necessary in order to ensure that the most important aspects of the Addon Operator service are maintained consistently by any available maintainer.
+
+
+# Maintenance Items
+
+The following are the aspects of the Addon Operator service that need maintenance. Some of them are either partially or fully maintained by automation such as a build job, or require human intervention.
+
+
+## Codebase
+
+The following are the properties of the Addon Operator codebase and its areas that need maintenance.
+
+**Type**: Opensource
+
+**License**: Apache-2.0
+
+**VCS**: <https://github.com/openshift/addon-operator>
+
+**Production Build**: <https://ci.int.devshift.net/job/openshift-addon-operator-gh-build-main/>
+
+**Production Branch**: main
+
+
+### Production Build
+
+Changes to the Addon Operator codebase are continuously integrated and its areas that need maintenance are the following.
+
+1. **Build Health**
+
+   1. The Addon Operator build job must be kept healthy at all times. 
+   2. Build failures must be addressed as soon as possible to avoid development or release interruptions. 
+   3. The Addon Operator build job alerts on build failures by sending these alerts to the slack channel #sd-mt-sre-alert
+   4. Most importantly, the Addon Operator codebase must be shippable at all times when it needs to be shipped.
+
+
+### Pull Requests
+
+Changes to the Addon Operator codebase are manually reviewed and approved by designated [reviewers and approvers](https://github.com/openshift/addon-operator/blob/main/OWNERS). Its areas that need maintenance are the following.
+
+1. **Merge Quality**
+
+   1. Every pull request must be reviewed and approved by a designated approver at all times before it is merged to main to help maintain the quality of the production codebase.
+
+
+### Third-Party Packages
+
+The Addon Operator codebase uses third-party packages; its areas that need maintenance are the following.
+
+1. **Package Upgrades**
+
+   1. Third-party packages used by the Addon Operator must be upgraded as soon as possible to primarily avoid potential problems inherited by the current state of such packages.
+   2. Any security-vulnerable packages must be upgraded or mitigated as soon as possible to avoid such vulnerabilities in the Addon Operator service.
+
+
+### Test Coverage
+
+The Addon Operator codebase is continuously tested by running a set of test suites that is part of the codebase itself. Its areas that need maintenance are the following.
+
+1. **Unit Tests**
+
+   1. The Addon Operator codebase must at least meet an 80% code coverage.
```

Comment:
80% is a bit overkill, let's dial it down to 70%, concrete result will follow once we have integrated Codecov.

```suggestion
   1. The Addon Operator codebase must at least meet an 70% code coverage.
```